### PR TITLE
Force emonhub log folder to user:root ownership

### PR DESCRIPTION
**Pull request by borpin**
Old permissions are not acceptable to new logrotate package

#190

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -101,7 +103,7 @@ fi
 # Fix emonhub log file permissions
 if [ -d /var/log/emonhub ]; then
```
Comment (borpin) [inline install.sh:104]: If the folder doesn't exist - no action - this is wrong isn't it?

**Comment by borpin** [comment]
Suggest adding an `ExecStartPre` entry to force the creation of the log folder with `root:root` permissions

https://github.com/openenergymonitor/emonhub/blob/ebbaa96e71765f050177b0ccdf2c670b8aa992ca/service/emonhub.service#L15

File: install.sh (hunk 1 above)
Comment (TrystanLea) [reply install.sh:104]: Can you open a new issue for this?

**Comment by TrystanLea** [comment]
Do we need a new issue for ExecStartPre?

**Pull request merged**